components/widgets/resources/src/main/resources/templates/geneVariantMacros.vm

```diff
@@ -107,13 +107,13 @@ $doc.display($prop.getName(), 'view', $fakeObj )##
 #foreach ($prop in $variantObj.xWikiClass.properties)
 #if ($prop.getName() != 'genesymbol' && !$options.defaultProperties.contains($prop.getName()) && $variantObj.getProperty($prop.getName()))
 #set($rawValue = $variantObj.getProperty($prop.getName()).value)
-#set($isValueEmpty = $rawValue == '' || $rawValue == 'NA' || $rawValue == [])
+#set($isValueEmpty = $rawValue == '' || $rawValue == 'NA' || $rawValue == [] || !$!{rawValue})
```

Comment:
Are you adding a check for `false` or `null`? If `null`, then it's safer to change `$rawValue == ''` into `"$!{rawValue}" == ''`.